sky/data/storage.py

```diff
@@ -340,6 +340,8 @@ def __init__(self,
         assert mode in StorageMode
         self.sync_on_reconstruction = sync_on_reconstruction
 
+        self.force_delete = False
```

Comment:
Can we add a `# TODO(romilb, zhwu): This is a workaround to support storage deletion for spot. Once sky storage supports forced management for external buckets, this can be deprecated.`